wsdl/ver10/advancedsecurity/wsdl/advancedsecurity.wsdl

```diff
@@ -748,7 +759,7 @@
 				</xs:attribute>
 				<xs:attribute name="PKCS10ExternalCertificationWithRSA" type="xs:boolean">
 					<xs:annotation>
-						<xs:documentation>Indicates support for creating PKCS#10 requests for RSA keys and uploading the certificate obtained from a CA..</xs:documentation>
+						<xs:documentation>Deprecated - Indicates support for creating PKCS#10 requests for RSA keys and uploading the certificate obtained from a CA..</xs:documentation>
```

Comment:
As per ONVIF TLS Add On specification, "Device shall support certificate configuration as defined by each of the
"SelfSignedCertificateCreationWithRSA" and the "PKCS10ExternalCertificationWithRSA" capabilities". Now PKCS10ExternalCertificationWithRSA and SelfSignedCertificateCreationWithRSA parameters are deprecated, what is the affect of TLS Addon with these parameters deprecation?